dbsetup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+
+psql -d $USER -c "CREATE DATABASE todoapp_devel"
```

Comment:
This command fails on my machine without specifying `-U postgres` as `USER` variable is not set to any valid database name.

Reply:
Done!